less/_components/quicksilver/resources.less

```diff
@@ -0,0 +1,60 @@
+/*!
+ * Copyright (c) 2020 Eclipse Foundation, Inc.
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * Contributors:
+ *   Eric Poirier <eric.poirier@eclipse-foundation.org>
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+*/
+
+.resources {
```

Comment:
I am not convinced this is the best name for this component. It's a bit vague and I am mostly concerned that we might be using that elsewhere.

Can we rename it to something like .resource-item ?

I also think that this file should be moved under /less/_components/quicksilver

Reply:
If .resources is a group of .resources-summary then I would rename

.resources to .resource-group

 .resources-summary is fine but I would prefer  .resources-item

What do you think

Reply:
Yes makes sense. I'll make those changes.